python/cudf/cudf/options.py

```diff
@@ -215,9 +241,26 @@ def _integer_and_none_validator(val):
         \tValid values are True or False. Default is False.
         """
     ),
-    _make_contains_validator([False, True]),
+    _spill_validator,
 )
 
+
+_register_option(
+    "copy_on_write",
+    _env_get_bool("CUDF_COPY_ON_WRITE", False),
+    textwrap.dedent(
+        """
+        Default behavior of performing shallow copies.
```

Comment:
I think this sentence is redundant, the default is explained later.
```suggestion
```

Reply:
Done